bin/README.md

```diff
@@ -0,0 +1,27 @@
+#Scripts in `bin` and their purpose
+
+Programs run via `autogen.sh` (or the equivalent in CMake) are indicated.
+
+|Program|Purpose|
+|-------|-------|
+|`buildhdf5`|Convenience script to build HDF5 using the Autotools|
+|`checkapi`|Checks if public API calls are used in internal functions|
+|`checkposix`|Checks if C/POSIX calls are prefixed with `HD`|
```

Comment:
chkposix will go away soon, when the HD prefixes go away

Reply:
@derobins , how about changing the `Convenience script to build HDF5 using the Autotools` to `Builds HDF5 using the Autotools` to match the other rows that start with verbs?